PocketKit/Sources/PocketKit/MyList/SavedItemsList/SavedItemsListViewModel.swift

```diff
@@ -11,7 +11,7 @@ class SavedItemsListViewModel: NSObject, ItemsListViewModel {
     private let _events: PassthroughSubject<ItemsListEvent<ItemIdentifier>, Never> = .init()
     var events: AnyPublisher<ItemsListEvent<ItemIdentifier>, Never> { _events.eraseToAnyPublisher() }
 
-    let selectionItem: SelectionItem = SelectionItem(title: "Saves", image: .init(asset: .saves))
+    let selectionItem: SelectionItem = SelectionItem(title: "Saves", image: .init(asset: .saves), selectedView: SelectedView.archive)
```

Comment:
should this be .saves instead? seems switched with `ArchivedItemsListViewModel.swift`